styleguide/components/Type.tsx

```diff
@@ -10,16 +10,15 @@ export const styles = ({ fontFamily, fontSize, color }: Theme) => ({
   },
 });
 
-const KUBEVIRT_TYPES = KUBEVIRT_CONSTS.allTypes;
+const KUBEVIRT_TYPES = KUBEVIRT_CONSTS.kubevirtTypes;
 interface TypeProps extends JssInjectedProps {
   children: React.ReactNode;
 }
 
 function getGithubTypeHref(type: string): string {
-  return `https://github.com/kubevirt-ui/kubevirt-api/blob/main/kubevirt/models/${type.replace(
-    /\[\]/,
-    '',
-  )}.ts`;
+  return `https://github.com/kubevirt-ui/kubevirt-api/blob/main/${
```

Comment:
should https://github.com/kubevirt-ui/kubevirt-api/blob/main/ be used as CONST ?